contracts/universalSchemes/VestingScheme.sol

```diff
@@ -184,7 +185,7 @@ contract VestingScheme is UniversalScheme, ExecutableInterface {
         agreementsCounter++;
 
         // Log new agreement and return id:
-        emit NewVestedAgreement(agreementsCounter-1);
+        emit NewVestedAgreement(agreementsCounter-1, "");
```

Comment:
??? "" ??  not nice.
please revert or add other event without proposalId

Reply:
added an event `ProposedVestedAgreement`